pycbc/workflow/coincidence.py

```diff
@@ -334,24 +336,26 @@ def merge_single_detector_hdf_files(workflow, bank_file, trigger_files, out_dir,
         out += node.output_files
     return out
 
-def setup_trigger_fitting(workflow, insps, hdfbank, veto_file, veto_name):
+def setup_trigger_fitting(workflow, insps, hdfbank, veto_file, veto_name,
+                          tags=None):
     if not workflow.cp.has_option('workflow-coincidence', 'do-trigger-fitting'):
         return FileList()
     else:
-        assert len(hdfbank) == 1  # must be a list with exactly 1 bank file
-        assert len(veto_file) == 1
-        assert len(veto_name) == 1
         smoothed_fit_files = FileList()
         for i in workflow.ifos:
             ifo_insp = [insp for insp in insps if (insp.ifo == i)]
             assert len(ifo_insp)==1
-            raw_node = PyCBCFitByTemplateExecutable(workflow.cp,
-                'fit_by_template', ifos=i).create_node(ifo_insp[0], hdfbank[0],
-                                                    veto_file[0], veto_name[0])
+            raw_exe = PyCBCFitByTemplateExecutable(workflow.cp,
+                                                   'fit_by_template', ifos=i,
+                                                   tags=tags)
+            raw_node = raw_exe.create_node(ifo_insp[0], hdfbank,
```

Comment:
ifo_insp is a list with single item and only used here - for neatness it may be best to initialise with a [0] at the end of the list comprehension?

Reply:
Changed as suggested.